UQ-eLIPSE/crucible-components
# Story/VETHUB-14

In this ticket, I've started constructing the data props and for this case it is the title.

Create `question-data.json` file which will expand throughout the project but any suggestions would be helpful.

Also created a `@data/*` alias so that we avoid using `../..`.

To test:

- Run `yarn dev` and then navigate to `https://localhost:5173` 
- Verify the title matches with the `question-data.json` title.
- Run `yarn test` to run the updated tests.
